index.html

```diff
@@ -37,190 +44,191 @@ <h2>Software Developer</h2>
+              no accounts or sign-ups required. has been the <br>
+              industry's standard dummy text ever since the <br>
+              1500s, when an unknown printer took a standard 
+              dummy text.</p>
+            <ul class="programming-languages">
+              <li>JS</li>
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>Ruby</li>
+            </ul> <button type="button" class="btn" id="1">See project</button>
+          </div>
+        </div>
+        <div class="work-container">
+          <div class="project-img"></div>
+          <div class="project-details">
+            <h2>Multi-post stories</h2>
+            <p>A daily selection of privately personalized reads;<br>
+              no accounts or sign-ups required. has been the <br>
+              industry's standard dummy text ever since the <br>
+              1500s, when an unknown printer took a standard 
+              dummy text.</p>
+            <ul class="programming-languages">
+              <li>JS</li>
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>Ruby</li>
+            </ul> <button type="button" class="btn" id="1">See project</button>
+          </div>
+        </div>
+        <div class="work-container">
+          <div class="project-img"></div>
+          <div class="project-details">
+            <h2>Multi-post stories</h2>
+            <p>A daily selection of privately personalized reads;<br>
+              no accounts or sign-ups required. has been the <br>
+              industry's standard dummy text ever since the <br>
+              1500s, when an unknown printer took a standard 
+              dummy text.</p>
+            <ul class="programming-languages">
+              <li><a href="">CSS</a></li>
+              <li><a href="">html</a></li>
+              <li><a href="">bootstrap</a></li>
+              <li><a href="">Ruby</a></li>
+            </ul> <button type="button" class="btn" id="1">See project</button>
+          </div>
+        </div>
+
+      </section>
+
+
+      <section class="contact-form">
+        <h2 class="contact-us">Let’s chat!</h2>
+        <p class="paragraph3">If you have an application you are
+          interested in developing, a feature that you <br>
+          need built or a project that needs coding.
+          I'd love to help with it </p>
+          <ul>
+            <li class="full name">
+              <input type="text" id="jeremy" name="user_name" />
+           <!-- </li>
+            <li> -->
+              <input type="text" id="lastname" name="last_name" />
+            </li>
+
+            <li>
+              <input type="email" id="mail" name="user_email" />
+            </li>
+            <li>
+              <textarea id="msg" name="user_message"></textarea>
+            </li>
+          </ul>
+
+          <li class="button">
+            <button type="button" class="btn" id="1">Send</button>
+          </li>
+        </form>
```

Comment:
Please note that your input fields should be wrapped by a `form` element,  instead of an `ul` element 😉. This will make your code semantically accurate, make your styling easier, and improve your code functionality.
To do so, kindly remove the `ul` and the `li` elements, you could have something like this:
```html
<form>
    <input type="text" id="jeremy" name="user_name" />
    <input type="text" id="lastname" name="last_name" />
    <input type="email" id="mail" name="user_email" />
    <textarea id="msg" name="user_message"></textarea>
    <button type="button" class="btn" id="1">Send</button>
</form>
```